modules/gdscript/register_types.cpp

```diff
@@ -139,6 +139,7 @@ static void _editor_init() {
 void initialize_gdscript_module(ModuleInitializationLevel p_level) {
 	if (p_level == MODULE_INITIALIZATION_LEVEL_SERVERS) {
 		GDREGISTER_CLASS(GDScript);
+		GDREGISTER_ABSTRACT_CLASS(GDScriptNativeClass);
```

Comment:
As far as I know, `GDScriptNativeClass` is not supposed to be exposed.

Reply:
This is more of a hack due to the lack of first-class types. `GDScriptNativeType` instances are obtainable, you can use them like `Node.new()` and `is_instance_of(object, Node)`, but they were never meant to be exposed.